robottelo/ssh.py

```diff
@@ -223,22 +263,52 @@ def upload_file(local_file, remote_file, key_filename=None, hostname=None):
     :param local_file: either a file path or a file-like object to be uploaded.
     :param remote_file: a remote file path where the uploaded file will be
         placed.
+    :param hostname: target machine hostname. If not provided will be used the
+        ``server.hostname`` from the configuration.
+    :param str key_filename: The path of the ssh private key to use when
+        connecting to the server. If it is ``None`` ``key_filename`` from
+        configuration's ``server`` section will be used.
+    """
+
+    with get_sftp_session(hostname, key_filename) as sftp:
+        _upload_file(sftp, local_file, remote_file)
+
+
+def upload_files(local_dir, remote_dir, file_search="*.txt", hostname=None):
```

Comment:
perhaps we also need the  key_filename also?

Reply:
Hmm, Yes I will update that.